src/categorical_algebra/FinCats.jl

```diff
@@ -716,11 +836,18 @@ end
 
 # Something like this should be built into Julia, but unfortunately isn't.
 
+"""
+Map two given functions across the respective keys and values of a dictionary.
+"""
```

Comment:
Thanks for writing docstrings for these helper functions! Hopefully we can find a way to get rid of them one day.